lib/active_merchant/billing/gateways/netbanx.rb

```diff
@@ -0,0 +1,245 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class NetbanxGateway < Gateway
+      # Netbanx is the new REST based API for Optimal Payments / Paysafe
+      self.test_url = 'https://api.test.netbanx.com/'
+      self.live_url = 'https://api.netbanx.com/'
+
+      self.supported_countries = ['CA', 'US', 'GB']
+      self.default_currency = 'CAD'
+      self.supported_cardtypes = [:visa, :master, :american_express, :discover]
```

Comment:
@bruno, why did you set these defaults for the countries, currencies and card types? I looked through the API documentation and it seems there are many more supported cards. https://developer.optimalpayments.com/en/documentation/card-payments-api/card-object/